src/modules/auth/webauthn/WebAuthnScreen.js

```diff
@@ -181,6 +195,25 @@ function WebAuthnCard(props) {
 		}
 	}
 
+	// Edit keyname
+	const onSubmitKeyName = async (values) => {
+		let response;
+		try {
+			response = await SeaCatAuthAPI.put(`/public/webauthn/${values.id}`, {"name": `${values.name}`});
+			// TODO: enable validation, when ready in SA service
+			if (response.data.result != 'OK') {
+				throw new Error(t("WebAuthnScreen|Something went wrong, can't change authenticator"));
```

Comment:
@Aringoaway You are not changing authenticator, but the `authenticator name`. Pls update it everywhere...